src/main/java/ca/corefacility/bioinformatics/irida/ria/web/models/ModelKeys.java

```diff
@@ -0,0 +1,19 @@
+package ca.corefacility.bioinformatics.irida.ria.web.models;
+
+/**
+ * Use as prefixes when generating UI model keys.  This will add a unique prefix to your model.
+ */
+public enum ModelKeys {
+	IridaBase("irida-base"), // THIS IS ONLY FOR TESTING PURPOSES!
```

Comment:
what do you mean only for testing purposes?

Reply:
This is there because I wanted to test the base class `IridaBase`, this class creates a key whenever a subclass is created based on a lookup in this enum based on the instantiated classes name (see line 17 of IridaBase.java).  Without having this key, the test would fail (see https://github.com/phac-nml/irida/pull/1311/files#diff-776d4ca8e17c24a1d0917798a3cbf3559f874e8f18363c53d6ec7b8694d165ac).